### PR TITLE
Fixed audio bugs

### PR DESCRIPTION
**Pull request by masonachu**
Fixed audio bugs:
- radio does not destroy when player goes back to main menu
- tutorial music does not destroy when player goes back to main menu

Additions:
- rebalanced audio in kitchen mission
- audio track readded to kitchen mission

### CODE REVIEW

File: Assets/Scripts/Audio/FMODRadio.cs [hunk 1]
```diff
@@ -16,17 +14,16 @@ void Start()
     {
         interactable = Utils.GetRequiredComponent<Interactable>(this);
         interactable.OnInteractEnd += ChangeStation;
-        musicEv = FMODUnity.RuntimeManager.CreateInstance(Music);
-        musicEv.set3DAttributes(FMODUnity.RuntimeUtils.To3DAttributes(transform.position)); 
-        musicEv.start();
+        musicEv = GetComponent<FMODUnity.StudioEventEmitter>();
```
Comment (chadleytan) [inline Assets/Scripts/Audio/FMODRadio.cs]: Should probably do `Utils.GetRequiredComponent()` like you did with interactable above (Alex always got me good with this so now I know loool)

Comment (masonachu) [reply Assets/Scripts/Audio/FMODRadio.cs]: Done!

**Review by jurcaua — APPROVED** [review]
i am SCARE of MAIN merging (for me, for my own PR) so if you don't have any actual changes in there, keep this at bay until my change is in possibly? lots delicate of stuff going on my side so might be safer that way
otherwise, WOW AM EXCITE FOR ALL THIS thank u sir <3

**Comment by masonachu** [comment]
> i am SCARE of MAIN merging (for me, for my own PR) so if you don't have any actual changes in there, keep this at bay until my change is in possibly? lots delicate of stuff going on my side so might be safer that way
> otherwise, WOW AM EXCITE FOR ALL THIS thank u sir <3

happy to hold off! easy to put back together woop

**Pull request closed without merging**